Week3/homework/2-step3.js

```diff
@@ -1,8 +1,27 @@
 'use strict';
 
-function threeFive(startIndex, stopIndex, threeCallback, fiveCallback) {
-  const values = [];
-  // Add your code here
-}
+{
+  function threeFive(startIndex, stopIndex, threeCallback, fiveCallback) {
+    const values = [];
+    for (let i = startIndex; i <= stopIndex; i++) {
+      values.push(i);
+    }
+
+    values.forEach(num => {
```

Comment:
A single element of a `values` array is `value`. I would prefer that name over the name `num`.